frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-status-card.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { VmStatus } from 'kubevirt-web-ui-components';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import HealthBody from '@console/shared/src/components/dashboard/health-card/HealthBody';
+import { DashboardItemProps } from '@console/internal/components/dashboard/with-dashboard-resources';
+import { VMDashboardContext } from '../../vms/vm-dashboard-context';
+import { VMAlerts } from './vm-alerts';
+
+import './_vm-status-card.scss';
```

Comment:
FYI, `scss` files starting with `_` are partials - smaller bits of styling to be pulled (imported) together in another stylesheet. For example: `public/style.scss` importing `public/components/nav/_nav-header.scss`.

If a `scss` file relates to a specific React component and it's _not_ imported elsewhere (other than in given React component), it's not really a partial, and therefore its name shouldn't start with `_`, in my opinion.

I'm OK with the current change, we can improve this later for better consistency.

Reply:
Thank you for the explanation and spotting that. It's a leftover as there is ongoing discussion whether we should import scss from the components or keep building the single tree of imports via `style.scss`.

I will remove that but considering context, this this line will be gone in a week or so.

Reply:
@spadgett @christianvogt I thought the convention we want to converge to is having `scss` files per-component (`import './FooBar.scss'` executed as part of component's module) and use `public/style.scss` only for the base styling & PF overrides, is that correct?

Reply:
Yes that's correct (imo).